rpm/sailfish-secrets.spec

```diff
@@ -265,3 +291,14 @@ cp -R lib/Crypto/doc/html/* %{buildroot}/%{_docdir}/Sailfish/Crypto/
 %postun -n libsailfishcryptoplugin
 /sbin/ldconfig || :
 
+%post -n %{secretsdaemon}
+systemctl-user daemon-reload || :
+systemctl-user reload-or-try-restart %{secretsdaemon} || :
```

Comment:
Aren't these things problematic during OS updating secrets daemon is probably not even running?

Reply:
Note that service file has "Conflicts=shutdown.target jolla-actdead-charging.service"